Example/src/ScrollEventExample.tsx

```diff
@@ -19,10 +19,10 @@ function ScrollExample() {
     onScroll: (event) => {
```

Comment:
missing type of `event`?

Reply:
No, it should be defined via generic type, so it doesn't have to be typed on every handler function.